R/theme_gtsummary_roche.R

```diff
@@ -32,19 +33,52 @@ theme_gtsummary_roche <- function(set_theme = TRUE, font_size = NULL) {
       )
     )
 
+
   # use arial font for flextables ----------------------------------------------
   lst_theme$`as_flex_table-lst:addl_cmds` <-
     c(
       lst_theme$`as_flex_table-lst:addl_cmds`,
-      list(valign = rlang::expr(flextable::font(fontname = "Arial", part = "all")))
+      list(
+        user_added1 = rlang::expr(flextable::font(fontname = "Arial", part = "all")),
+        user_added1 = rlang::expr(flextable::fontsize(size = 7, part = "footer"))#,
+        # user_added1 = rlang::expr(
+        #   flextable::border_outer(part = "body", border = border) %>%
+        #     flextable::border_outer(part = "header", border = border) %>%
+        #     flextable::border(
+        #       part = "header", j = 1,
+        #       border.left = border,
+        #       border.right = border
+        #     ) %>%
+        #     flextable::border(
+        #       part = "header", j = 1, i = 1,
+        #       border.top = border
+        #     ) # %>%
+            # flextable::border(
+            #   part = "header", j = 1, i = flextable::nrow_part(flx, "header"),
+            #   border.bottom = border
+            # ) %>%
+            # flextable::border(
+            #   part = "header", j = seq(2, ncol),
+            #   border.left = border,
+            #   border.right = border
+            # )
+        # )
+      )
     )
+  # todo:
+  # * Bold label rows
+  # * single line spacing
+  # * vertical padding
+  # * margins
+  # (*) Page sizes defaults -> into docx save along with pagination (? or not)
+  # (*) flextable::fix_border_issues(flx) # Fixes some rendering gaps in borders
 
 
   # add a monospace font for gt ------------------------------------------------
   lst_theme$`as_gt-lst:addl_cmds` <-
     c(
       lst_theme$`as_gt-lst:addl_cmds`,
-      list(tab_spanner = rlang::expr(gt::opt_table_font(stack = "monospace-code")))
+      list(font_type = rlang::expr(gt::opt_table_font(stack = "monospace-code")))
```

Comment:
why the names are fixed?

Reply:
For some of the print engines, the order in which the additional commands are executed is important. These names indicate where to insert these names.

You can see the default code (along with their assigned names) with `as_flex_table(x, return_calls  = TRUE)`.